tests/experimental/datasources/data_asset/data_connector/test_s3_data_connector.py

```diff
@@ -293,7 +293,7 @@ def test_return_all_batch_definitions_unsorted():
 #         prefix="",
 #     )
 #     # noinspection PyProtectedMember
-#     my_data_connector._refresh_data_references_cache()
+#     my_data_connector._populate_data_reference_cache_if_empty()
```

Comment:
this update is likely the result of a find/replace, but should nonetheless be removed